### ISSUE TITLE
[pt] Idea for rule: “serve para” → “permite/possibilita” - 2023-03-15

### ISSUE THREAD

**Issue by marcoagpinto** [languagetool-org/languagetool#8047]
Hello @ricardojosehlima

Sorry for not opening rules tickets in GitHub but for two days that I have been feeling ill, I feel stoned, I hope it is not COVID-19 since I have all the vaccines.

Take a look at:
`Este botão serve para gravar ficheiros JPEG.`
TO
`Este botão permite gravar ficheiros JPEG.`

Is it a good idea?

Thanks!

**Comment by ricardojosehlima** [comment]
Hi! Best wishes for your health!
As for the rule, some cases to look at:
Ele não serve para esse serviço --> Ele não permite esse serviço (????)
Meu amigo serve como ajudante na empresa --> Meu amigo permite como ajudante (????)

The great problem of the rule is that it can't apply to animate nouns...

**Comment by marcoagpinto** [comment]
Ahhhh... it won't be implemented then, thank you!

**Comment by marcoagpinto** [comment]
ahhhh... and with a verb in front?

I was just wondering.

**Comment by marcoagpinto** [comment]
@ricardojosehlima 

Should I try with a verb in front to see what happens?

Thanks!

**Comment by ricardojosehlima** [comment]
Hi, didn't understand your question

**Comment by marcoagpinto** [comment]
Heya!

I am solid as a rock!

I was only stoned for two or three days, or so.

What I meant was:
`Este botão serve para gravar ficheiros JPEG.`

Your examples didn't work with the rule, but, will it work if we have a verb in front of the “para”?

This is what I was asking.

😛 

Thanks!

**Comment by ricardojosehlima** [comment]
Hi not so sure, these verbs carry relevant meanings

**Comment by marcoagpinto** [comment]
Ahhhh… I will do a test at 5am to see what happens.

If it doesn't work, it isn't much of a rule anyway.

Thanks!

😄 😄 😄 😄

**Comment by ricardojosehlima** [comment]
Looks good for me too!

**Comment by marcoagpinto** [comment]
Cool!

❤️ ❤️ ❤️ ❤️ ❤️ ❤️ ❤️ ❤️ ❤️ ❤️ ❤️ ❤️ ❤️ ❤️ ❤️ 

I will start working on the “-chave” rule in a few hours.

😛 😛 😛 😛 😛 😛 😛 😛 😛 😛 😛 😛 😛 😛 😛

**Comment by susanaboatto** [comment]
Still doesn't quite work when it comes to specifying function:

**o governo serve para governar.**
**Esta colher serve para tomar sopa.**

**Comment by marcoagpinto** [comment]
Yes, Susana, there is no good way of solving it, but it isn't perfect. 😋

**Comment by marcoagpinto** [comment]
@susanaboatto 

Maybe add an exception line for action verbs?

"governar, tomar, etc." and as verbs are found they are added there?
